neurom/features/morphology.py

```diff
@@ -63,6 +63,19 @@
 feature = partial(feature, namespace=NameSpace.NEURON)
 
 
+def _assert_soma_center(morph, feature_name):
+    if morph.soma.center is None:
+        morph_name = getattr(morph, "name", "")
+        if not morph_name:
+            morph_name = " "
+        else:
+            morph_name = f" '{morph_name}' "
```

Comment:
neurom Morphology always has a name, doesn't it?

Reply:
Yeah for the `getattr` I'm not sure, if we are sure that `morph` is a `neurom.core.Morphology` object, then we don't need the `getattr` indeed.
But for the value, it can be anything, like empty string or even None if the user assigned it. So in this case I thought we don't have to print it, but after more thoughts it can actually be an interesting information so I will always print it...

Reply:
But the `name` attribute is not read-only, so it can be anything if the user wants to change it.

Reply:
If the user explicitly mutates the name with None or "", then this is what they are getting in the error. :)